synthtool/languages/python.py

```diff
@@ -160,41 +163,54 @@ def py_samples(*, root: PathOrStr = None, skip_readmes: bool = False) -> None:
         s.copy([result], excludes=excludes)
 
 
-def configure_previous_major_version_branches() -> None:
-    """Configure releases from previous major version branches by editing
-    `.github/release-please.yml`.
+def get_library_version() -> packaging_version.Version:
+    """Returns the current library version.
 
     The current library version is obtained from `version.py` in `google/**/version.py`,
     or the `setup.py`.
 
-    Releases are configured for all previous major versions. For example,
-    if the library version is currently 3.5.1, the release-please config
-    will include v2, v1, and v0.
+    Raises:
+        RuntimeError: If no version string found.
     """
+    version_paths = list(Path(".").glob("google/**/version.py")) + [Path("setup.py")]
```

Comment:
Please could you add `"google/**/gapic_version.py"` as well or change this to `list(Path(".").glob("google/**/*version.py"))`

Reply:
Updated - please take a look.